runtime/util/romclasswalk.c

```diff
@@ -183,6 +183,17 @@ void allSlotsInROMClassDo(J9ROMClass* romClass,
 		}
 	}
 
+	/* walk enclosed inner classes SRPs block */
+	srpCursor = J9ROMCLASS_ENCLOSEDINNERCLASSES(romClass);
+	count = romClass->enclosedInnerClassCount;
+	rangeValid = callbacks->validateRangeCallback(romClass, srpCursor, count * sizeof(J9SRP), userData);
+	if (rangeValid) {
+		callbacks->sectionCallback(romClass, srpCursor, count * sizeof(J9SRP), "enclosedInnerClassesSRPs", userData);
+		for (; count > 0; count--) {
```

Comment:
`> 0` is a misleading test of an unsigned variable; please use `!= 0` instead.

Reply:
Fixed.

Reply:
Fixed.